tycho-plugins/target-platform-utils/pom.xml

```diff
@@ -34,5 +39,44 @@
     	<artifactId>tycho-p2-publisher-plugin</artifactId>
     	<version>${tychoVersion}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven.plugin-testing</groupId>
+      <artifactId>maven-plugin-testing-harness</artifactId>
+      <version>3.3.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.shared</groupId>
+      <artifactId>maven-verifier</artifactId>
+      <version>1.6</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-test</id>
+	         <configuration>
+	           <skip>true</skip>
+	         </configuration>
+          </execution>
+          <execution>
+            <id>it-tests</id>
```

Comment:
if these are integration tests, I think our convention is JBoss Tools is "ITests" rather than "it-tests"